src/Command.js

```diff
@@ -83,8 +96,8 @@ export default class Command {
   }
 
   runValidations() {
-    if (this.concurrency < 1) {
-      this.logger.warn("command must be run with at least one thread.");
+    if (!GitUtilities.isInitialized()) {
```

Comment:
Hi @evocateur,

Just testing the upgrade path from v1-beta.36 to v2-rc.4 and come across this check.  We noticed it as it started breaking on CI where it copies a previously cloned repo to another area to perform more tasks (rather than cloning again).

I'm sure we can change our build setup easy enough - but I was just wondering what the reason is as to why Lerna needs to be executed in a git repo or if this check could be dropped?

Thanks for your help